src/hooks/useAccounts.tsx

```diff
@@ -224,7 +243,17 @@ const useAccountsContext = () => {
 
```

Comment:
This code block is run when we have an evmAddress but no dydxWallet yet. This means the evm wallet is connected, but we have not derive the dydxWallet yet.